docs/en/week05/05-3.md

```diff
@@ -10,7 +10,7 @@ date: 28 Feb 2020
 
 In this part we will discuss convolution, since we would like to explore the sparsity, stationarity, compositionality of the data.
 
-Instead of using the matrix $A$ above, we will change the matrix width to the kernel size $k$. Therefore, each row of the matrix is a kernel. We can use the kernels by stacking and shifting (see Fig 1). Then we can have $m$ layers of height $n-k+1$.
+Instead of using the matrix $A$ discussed from previous week, we will change the matrix width to the kernel size $k$. Therefore, each row of the matrix is a kernel. We can use the kernels by stacking and shifting (see Fig 1). Then we can have $m$ layers of height $n-k+1$.
```

Comment:
```suggestion
Instead of using the matrix $A$ discussed in the [previou week](en/week04/04-1), we will change the matrix width to the kernel size $k$. Therefore, each row of the matrix is a kernel. We can use the kernels by stacking and shifting (see Fig 1). Then we can have $m$ layers of height $n-k+1$.
```